packages/common-all/src/service/ConfigService.ts

```diff
@@ -9,29 +9,32 @@ import * as YamlUtils from "../yaml";
 import { DConfigLegacy } from "../oneoff/ConfigCompat";
 
 export type ConfigServiceOpts = {
-  wsRoot: URI;
```

Comment:
Having a singleton ConfigService that deals with only one `wsRoot` doesn't work. We have places where we deal with multiple workspaces (e.g. workspace / seed creation or deletion related operations)

We still do want to keep the config service as a singleton unless we want to provide it a file store instance every time we use it.

In later iterations I'm planning to wrap the config service into an tsyringe @injectable and pass it around to places where we need to access configs instead of having it globally accessible to make it clear where we create the instance